doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,415 @@
+
+The manager should respond on the basis of matching any of the traits in
+the set, even if it doesn't understand many others, unless it explicitly
+needs to opt-out for some specialisation (not illustrated):
+
+```python
+if CacheTrait.kId in traitSet:
+    continue
+if FileTrait.kId in traitSet:
+    ManagedTrait.imbueTo(policy)
+```
+
+Such that it opts-in to managing all Entities with the file Trait in
+their set, aside from caches:
+
+```python
+policies = [
+    { 'managed': {...} },  # Main document
+    { 'managed': {...} },  # Generated images
+    { 'managed': {...} },  # Generated maps
+    {},                    # Temporary cache
+]
+```
+
+Any other queries for trait sets without the file trait would be empty
+`{}` indicating un-managed status.
+
+#### Pros
+
+- Hosts can query specific and precise trait sets, but the manager does
+  not need to worry about all possible permutations.
+
+#### Cons
+
+- There is no way for a host to determine what traits can be resolved
+  (or persisted) for any given entity, and so must tolerate missing data
+  at a later date (e.g. the color space for the `colorManaged` trait, if
+  the manager can't ever provide this).
+
+### Option 2 - Exact Matching
+
+The manager should respond only to sets where it is capable of resolving
+or persisting all of the requested trait set:
+
+```python
+if traitSet == { FileTrait.kId }:
+    ManagedTrait.imbueTo(policy)
+```
+
+Such that it _only_ opts into managing the specifically limited trait
+set that it can support:
+
+```python
+policies = [
+    {},  # Main document
+    {},  # Generated Images
+    {},  # Generated maps
+    {}   # Temporary caches
+]
+```
+
+The Host would receive empty responses (`{}`) for all of the original
+trait sets, and so must potentially decompose and re-try its queried
+trait sets to reveal some supported sub-set. For example, if ultimately
+it just needed the file path, it could try these combinations:
+
+```python
+traitSets = [
+    { 'file', 'nodeGraph' },
+    { 'file', 'image', 'colorManaged' },
+    { 'file', 'image' },
+    { 'file', 'image', 'latLong', 'colorManaged' },
+    { 'file', 'image', 'latLong' },
+    { 'file', 'cache' },
+    { 'file' }
+]
+```
+
+Resulting in:
+
+```python
+policies = [
+    {},
+    {},
+    {},
+    {},
+    {},
+    {},
+    { 'managed': {...} }
+]
+```
+
+#### Pros
+
+- Conveys only what traits are supported to the host, allowing
+  behaviour to be properly adapted to the manager's capabilities.
+
+#### Cons
+
+- No way to know that actually the manager was interested in managing
+  all but one of the requested trait sets.
+- Host implementation is significantly more complicated.
+- No way to hint at any additional traits that may be resolvable
+  outside the typing trait set.
+
+### Option 3 - Supported Traits Response
+
+The host should also include any additional traits an entities trait set
+that it may attempt to resolve in the future, further specialising it.
```

Comment:
[fix] Not sure of the wording here
```suggestion
The host should also include any additional traits from an entity's trait set
that it may attempt to resolve in the future, further specialising it.
```
or
```suggestion
The host should also include any additional entity traits
that it may attempt to resolve in the future, further specialising it.
```